bats/config.bats

```diff
@@ -164,3 +164,22 @@ teardown() {
     regex='John Doe <john@doe.com>'
     [[ "$output" =~ "$regex" ]] || false
 }
+
+@test "sql-server: starting test without server throws error" {
```

Comment:
This test name makes no sense? How about?

`sql-server: dolt sql commit() function errors appropriately when user.name or user.email is unset`